pkgs/tools/security/ripasso/cursive.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, ncurses, python3, openssl, libgpgerror, gpgme, xorg }:
+
+with rustPlatform;
+buildRustPackage rec {
+
+  version = "b64b1d9b697b2246b96db24a9d9ae571eab839aa";
```

Comment:
Version should not be a commit id it should be something along the lines `unstable-<commit-date>`